### PR TITLE
add neontv.co.nz status

### PR DESCRIPTION
**Pull request by kety-folf**
[images showing code works](https://drive.google.com/drive/folders/1Iit_jzxp0CibLtJ33cr936xlW_hjhWQs?usp=sharing)

### CODE REVIEW

File: websites/N/Neon/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    "author": {
+      "name": "Kety The Folf",
+      "id": "263443630767734784"
+    },
+    "contributors": [
+        {
+          "name": "SlowLife",
+          "id": "374905512661221377"
+        }
+      ],
```
Comment (ririxidev) [inline websites/N/Neon/dist/metadata.json:13]: No contributors on release.

File: websites/N/Neon/presence.ts [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "key",
+    details: "Browsing...",
+    startTimestamp: timestamp
+  },
+  pathname = document.location.pathname;
+
+  switch(true) {
```
Comment (ririxidev) [inline websites/N/Neon/presence.ts:20]: Please don't use switch(true) statement.

File: websites/N/Neon/presence.ts [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+      }
+      break;
+
+    case document.location.search.startsWith("?"):
+      presenceData.details = "Searching for:";
+      presenceData.state = (new URLSearchParams(document.location.search)).get("search");
+      break;
+  }
+
+  if (!(await presence.getSetting("buttons")))
```
Comment (ririxidev) [inline websites/N/Neon/presence.ts:117]:
```suggestion
  if (!(await presence.getSetting("buttons")) && presenceData.buttons)
```

**Review by ririxidev — CHANGES_REQUESTED** [review]

File: websites/N/Neon/dist/metadata.json (hunk 1 above)
Comment (Bas950) [reply websites/N/Neon/dist/metadata.json:13]: Nah SlowLife actually helped Kety a lot, I saw in #presence-dev, its okay this time.

File: websites/N/Neon/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    "thumbnail": "https://i.imgur.com/dO4SNtS.png",
+    "version": "1.0.0",
+    "color": "#000000",
+    "tags": ["videos", "streams"],
+    "url": [
+      "www.neontv.co.nz",
+      "neontv.co.nz"
+    ],
+    "description": {
+      "en": "Neon, marketed as NEON until 2020, is a subscription video on demand service offering a range of television shows and movies. The platform is a division of Sky Network Television Limited."
```
Comment (Bas950) [inline websites/N/Neon/dist/metadata.json:25]:
```suggestion
      "en": "Watch hit TV shows on-demand and rent blockbuster movies with NEON. No term contracts."
```
Use the official description.

File: websites/N/Neon/presence.ts (hunk 2 above)
Comment (Bas950) [reply websites/N/Neon/presence.ts:20]: Why not?

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by ririxidev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**